calico_containers/pycalico/datastore.py

```diff
@@ -113,6 +123,51 @@ def wrapped(*args, **kwargs):
     return wrapped
 
 
+def assert_valid_etcd_keys(scheme, key, cert, ca_cert):
```

Comment:
Strictly speaking this is not asserting (which would be if it raised an AssertionError).  Probably should be called validate_etcd_keys or something similar.

Also, the interface and messages don't really marry correctly.  The interface has you pass in the keys and certificates, but the error messages describe the environment variable names.

I wonder - should this simply be part of the Datastore connection processing?